Tools/Gulp/tasks/gulpTasks-librariesES6.js

```diff
@@ -9,6 +9,7 @@ var symlinkDir = require('symlink-dir');
 // Gulp Helpers
 var rmDir = require("../../NodeHelpers/rmDir");
 var processImports = require("../helpers/gulp-processImportsToEs6");
+var convertImportsToJs = require("../helpers/gulp-importFromJsFiles");

```

Comment:
Could this be merged with the processImports step to prevent going over all the file twice assuming the steps is taking some times (if not, please fully disregard :-)) ?

Reply:
that was my first attempt. However it will not work without changing the webpack configuration that builds inspector etc', since it changes the sources being used. This is why I wanted to do that on the dist files of projects not built with webpack.